src/lib.rs

```diff
@@ -95,6 +96,10 @@ fn main(mut cx: ModuleContext) -> NeonResult<()> {
     cx.export_function("in_memory_smt_update", InMemorySMT::js_update)?;
     cx.export_function("in_memory_smt_prove", InMemorySMT::js_prove)?;
     cx.export_function("in_memory_smt_verify", InMemorySMT::js_verify)?;
+    cx.export_function(
```

Comment:
Just for the better reading and view, it would be better to define a new variable like `in_memory_smt_new` for `InMemorySMT::js_calculate_root`